### DISCUSSION TITLE
Selecting Operator  twice making it null after some time

### DISCUSSION THREAD

**Discussion by kuljeet2604** [react-querybuilder/react-querybuilder#776] [General]
PFA screenshot : 
1 : when we click operator for the first time
2 : when we click/select the same operator for the second time 

PFA sandbox: issue is with when arity is "unary", selecting the operator for the second time make it "" and the 3rd/value  box appears.
can be replicated in the sandbox link.
https://codesandbox.io/p/devbox/interesting-hamilton-swc7nv

**Comment by jakeboone02** [comment]
This is tracked by #781, will be fixed by #783, and will be included in v7.7.0 which should be released soon.
